react/features/base/tracks/middleware.js

```diff
@@ -108,30 +106,23 @@ MiddlewareRegistry.register(store => next => action => {
             const participantID = jitsiTrack.getParticipantId();
             const isVideoTrack = jitsiTrack.isVideoTrack();
 
-            if (jitsiTrack.isLocal()) {
-                if (isVideoTrack) {
+            if (isVideoTrack) {
+                if (jitsiTrack.isLocal()) {
                     APP.conference.setVideoMuteStatus(muted);
                 } else {
-                    APP.conference.setAudioMuteStatus(muted);
+                    APP.UI.setVideoMuted(participantID, muted);
                 }
-            }
-
-            if (isVideoTrack) {
-                APP.UI.setVideoMuted(participantID, muted);
                 APP.UI.onPeerVideoTypeChanged(
                     participantID,
                     jitsiTrack.videoType);
             } else {
-                APP.UI.setAudioMuted(participantID, muted);
+                // eslint-disable-next-line no-lonely-if
```

Comment:
This will be much harder to read, because first if splits the code on audio and video, otherwise you have 3 chained else if. Much less readable at least for me, but I can change if you insist.